src/lecture_feedback/user_session.py

```diff
@@ -0,0 +1,26 @@
+import uuid
+
+import streamlit as st
+
+
+class UserSession:
```

Comment:
Should we call this something like `StreamlitTab` for disambiguation with our other session names?

Also, the `User` part initially confused me and I had to carefully look at code to find out what the difference between `UserSession` and `UserStatsTracker` is